### PR TITLE
Update PR Template

### PR DESCRIPTION
**Pull request by raisedadead**
New contributors left spaces in the `[ x]` which did not render properly, rather having the contributors click on the check boxes is a better option, since GitHub now supports it.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -5,23 +5,23 @@
 <!-- Make sure that your PR is not a duplicate -->
 
 #### Pre-Submission Checklist
-<!-- Go over all points below, and put an `x` in all the boxes that apply. -->
-<!-- All points should be checked, otherwise, read the CONTRIBUTING guidelines from above-->
+<!-- Go over all points below, and after creating the PR click on all the check boxes that apply. -->
```
Comment (atjonathan) [inline .github/PULL_REQUEST_TEMPLATE.md]: ~~, and after creating the PR click on all~~ , and after creating the PR, click on all

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -5,23 +5,23 @@
...
+<!-- All points should be verified, otherwise, read the CONTRIBUTING guidelines from above-->
 <!-- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
 - [ ] Your pull request targets the `staging` branch of FreeCodeCamp.
 - [ ] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
 - [ ] You have only one commit (if not, [squash](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-rebase#squashing-multiple-commits-into-one) them into one commit).
 - [ ] All new and existing tests pass the command `npm run test-challenges`. Use `git commit --amend` to amend any fixes.
 
 #### Type of Change
-<!-- What type of change does your code introduce? Put an `x` in the box that applies. -->
+<!-- What type of change does your code introduce? After creating the PR click on all the check boxes that apply. -->
```
Comment (atjonathan) [inline .github/PULL_REQUEST_TEMPLATE.md]: ~~After creating the PR click on all the check boxes that apply.~~ After creating the PR, click on all the **checkboxes** that apply.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -5,23 +5,23 @@
...
 - [ ] Small bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds new functionality)
 - [ ] Breaking change (fix or feature that would change existing functionality)
 - [ ] Add new translation (feature adding new translations)
 
 #### Checklist:
-<!-- Go over all points below, and put an `x` in all the boxes that apply. -->
+<!-- Go over all points below, and after creating the PR click on all the check boxes that apply. -->
```
Comment (atjonathan) [inline .github/PULL_REQUEST_TEMPLATE.md]: ~~after creating the PR click on all the check boxes that apply.~~ after creating the PR, **tick** the **checkboxes** that apply.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -5,23 +5,23 @@
 <!-- Make sure that your PR is not a duplicate -->
 
 #### Pre-Submission Checklist
-<!-- Go over all points below, and put an `x` in all the boxes that apply. -->
-<!-- All points should be checked, otherwise, read the CONTRIBUTING guidelines from above-->
+<!-- Go over all points below, and after creating the PR, click on all the checkboxes that apply. -->
```
Comment (atjonathan) [inline .github/PULL_REQUEST_TEMPLATE.md]: ~~click on all the checkboxes that apply.~~ tick all the checkboxes that apply.

**Sorry** :disappointed:, missed this one :joy:

Comment (raisedadead) [inline .github/PULL_REQUEST_TEMPLATE.md]: Common man.......! :sweat_smile:

**Comment by QuincyLarson** [comment]
@raisedadead how common is the leading/trailing space issue? 

Contributors should be checking these boxes as they prep the PR. I feel like asking them to first create the PR, then to go back through and check the boxes will result in them cancelling PRs once they realize they didn't do something, or labeling more PRs WIP, etc.

**Comment by atjonathan** [comment]
@QuincyLarson, sorry to answer something that's not my question, but the issue is very common.

**Comment by raisedadead** [comment]
@QuincyLarson yeah, as @atjonathan mentioned already its pretty common, GitHub's feature is rather neat, while the markdown syntax just confuses new contributors.

**Comment by atjonathan** [comment]
#### **LGTM :+1:**
Thanks ninja :tada:

**Pull request merged**